lib/util.js

```diff
@@ -35,5 +35,9 @@ module.exports = {
     return crc(JSON.stringify(sess));
   },
 
+  isNil(val) {
+    return val === null || typeof val === 'undefined';
```

Comment:
This modification is a little bit over abstract. I'd like to maintain the original style. :(

Reply:
Ok